pkg/operator/handlers/dapr_handler.go

```diff
@@ -16,17 +15,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 
+	"github.com/dapr/dapr/pkg/injector/annotations"
 	"github.com/dapr/dapr/pkg/operator/monitoring"
 	"github.com/dapr/dapr/pkg/validation"
 	"github.com/dapr/dapr/utils"
 	"github.com/dapr/kit/logger"
 )
 
 const (
-	daprEnabledAnnotationKey        = "dapr.io/enabled"
```

Comment:
Reviewers: these constants were already defined in pkg/injector/annotations so importing them from there instead